packages/editor/src/components/document-tools/index.js

```diff
@@ -23,51 +23,58 @@ import { store as keyboardShortcutsStore } from '@wordpress/keyboard-shortcuts';
 /**
  * Internal dependencies
  */
-import { store as editPostStore } from '../../../store';
-import { unlock } from '../../../lock-unlock';
+import { unlock } from '../../lock-unlock';
+import { store as editorStore } from '../../store';
+import EditorHistoryRedo from '../editor-history/redo';
+import EditorHistoryUndo from '../editor-history/undo';
 
 const { useCanBlockToolbarBeFocused } = unlock( blockEditorPrivateApis );
 
 const preventDefault = ( event ) => {
 	event.preventDefault();
 };
 
-function HeaderToolbar( { hasFixedToolbar } ) {
+function DocumentTools( {
+	className,
+	showIconLabels,
+	disableBlockTools = false,
+	children,
+} ) {
 	const inserterButton = useRef();
 	const { setIsInserterOpened, setIsListViewOpened } =
 		useDispatch( editorStore );
 	const {
-		isInserterEnabled,
+		isInserterDisabled,
 		isInserterOpened,
-		isTextModeEnabled,
-		showIconLabels,
 		isListViewOpen,
 		listViewShortcut,
 		listViewToggleRef,
+		hasFixedToolbar,
 	} = useSelect( ( select ) => {
-		const { hasInserterItems, getBlockRootClientId, getBlockSelectionEnd } =
-			select( blockEditorStore );
+		const {
+			hasInserterItems,
+			getBlockRootClientId,
+			getBlockSelectionEnd,
+			getSettings,
+		} = select( blockEditorStore );
```

Comment:
Aren't settings also available in the store of the package? I guess we anyway add a subscription to the block editor store so it's fine